src/components/Avatar/Avatar.visual.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Avatar } from './';
+
+class AvatarVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    return <Avatar data-hook={'storybook-e2e-Avatar'} {...this.props} />;
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
```

Comment:
We should render the three types of Avatars, with the different sizings for the visual tests.
You can take a look at other `*.visual` files as a reference.

Reply:
Added visual tests for all the scenarios;